classes/event/poll_conversion_status.php

```diff
@@ -46,6 +46,6 @@ public static function get_name() {
     }
 
     public function get_description() {
-        return get_string('event:poll_conversion_status', 'fileconverter_librelambda');
```

Comment:
Should we remove lang strings then or update them to be able to use id and status?

Reply:
Ah ok. I see it's been used in other place/.

Reply:
This string is used for get_name() method so it shouldn't be removed.
Normally, lang strings are not used in get_description() methods